ip_country/src/country_finder.rs

```diff
@@ -123,7 +84,7 @@ mod tests {
         )
     }
 
-    pub(crate) fn ipv6_country_data() -> (Vec<u64>, usize) {
+    pub fn ipv6_country_data() -> (Vec<u64>, usize) {
```

Comment:
Ehhhh. 

You need to make a choice. Either keep these two functions we can see here or work only with file `ip_country/test_dbip_country`. I don't like you have created two pairs of these test substitutions. Given you're already using the trick with mutually exclusive, overriding conditional compilation attributes at the beginning of this whole file, I might go that way rather and delete this. However, I want you to discuss the time measure for these test. Do we need to check the real times in order to keep a track of how much it can take the Node to finish the boostrap?  If we need such test, you CAN'T use the aforementioned technique with the `cfg` attributes because you'd need both the short-handed and the full-range datasets!

Reply:
I do not understand this comment. We can discuss this on the call.